client/src/components/Stripe/Checkout.js

```diff
@@ -42,6 +52,16 @@ const Checkout = ({ isCartEmpty, activeStep, steps }) => {
       history.push("/error");
     }
   };
+  const openSnack = () => {
+    enqueueSnackbar("Please signing or signup to proceed with payment", {
+      variant: "info",
```

Comment:
Only comment is that "signing" should be "signin". The same for line 72 in Explore page.

Reply:
> Only comment is that "signing" should be "signin". The same for line 72 in Explore page.

Thanks ;)